masq_lib/Cargo.toml

```diff
@@ -20,6 +21,7 @@ serde_derive = "1.0.99"
 serde_json = "1.0.40"
 tiny-hderive = "0.2.1"
 toml = "0.5.3"
+web3 = {version = "0.8.0", default-features = false}
```

Comment:
will this stay `0.8.0` or update to `0.11.0`?

Reply:
I will leave it. Because in this card there is no need to touch it. We know that I will but it will be in another card, hopefully after we complete this one.

But yes the change to 0.11.0 is planned.